docs/eventing/flows/sequence/README.md

```diff
@@ -3,6 +3,10 @@
 Sequence CRD provides a way to define an in-order list of functions that will be
 invoked. Each step can modify, filter or create a new kind of an event. Sequence
 creates `Channel`s and `Subscription`s under the hood.
+    !!! info
+        Sequence needs "hairpin" traffic. Please verify that your pod can reach itself via the service IP.
```

Comment:
```suggestion
creates `Channel`s and `Subscription`s under the hood.

    !!! info
        Sequence needs "hairpin" traffic. Please verify that your pod can reach itself via the service IP.
```

(otherwise info box doesn't show up)